tsconfig.json

```diff
@@ -22,16 +22,15 @@
     // modules
     "baseUrl": "src/",
     "paths": {
-      "Components/*": ["Components/*"],
+      "Components/*": ["components/*"]
```

Comment:
Why?
wont this break the code

Reply:
If I don't make these changes , code is showing errors in my IDE.